stream-chat-android-client/src/main/java/io/getstream/chat/android/client/models/Message.kt

```diff
@@ -87,6 +87,11 @@ public data class Message(
      */
     var syncStatus: SyncStatus = SyncStatus.COMPLETED,
```

Comment:
Do we need to have two sources of information? `MessageSyncDescription` also contains these information

Reply:
I currently treat `MessageSyncDescription` as an addition to `SyncStatus`, because `SyncStatus` is more generic and it's used in other models like `Reaction` and `Channel`. 
So Idea was to make a more descriptive model for messages.
- If `SyncStatus` is `IN_PROGRESS` then `MessageSyncDescription` might be in `IN_PROGRESS_AWAIT_ATTACHMENTS`.
- If `SyncStatus` is `FAILED` then `MessageSyncDescription` might be in `FAILED_MODERATION`.

Moreover, we cannot just delete `SyncStatus` or  remove any statuses from it, because it's gonna be a breaking change